.github/workflows/autogenerated-files.yml

```diff
@@ -24,6 +24,10 @@ jobs:
       - name: Set up Homebrew
         id: set-up-homebrew
         uses: Homebrew/actions/setup-homebrew@master
+        with:
+          core: true
+          cask: true
+          test-bot: true
```

Comment:
Don't think core and cask are needed here - we don't generate manpages from them. They weren't before since this is Linux.